src/transformState.js

```diff
@@ -5,7 +5,30 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'clear':
+        for (const key in state) {
+          delete state[key];
+        }
+        break;
+
+      case 'removeProperties':
+        for (const key in state) {
+          if (action.keysToRemove.includes(key)) {
+            delete state[key];
+          }
```

Comment:
You may iterate not the state but `keysToRemove`
```suggestion
        for (const keyToRemove of action.keysToRemove) {
            delete state[keyToRemove];
        }